### ISSUE TITLE
finetuning

### ISSUE THREAD

**Issue by dariospin93** [debidatta/pixelobjectness-bilinear#1]
Hi, I would like to try a finetuning of your network on my dataset to see if it improves the performances on my test set.
could you please share the train prototxt? It will be great!

thank you in advance
